packages/core/lib/integration/snapshot-store/in-memory.snapshot-store.ts

```diff
@@ -24,7 +24,7 @@ export type InMemorySnapshotEntity<A extends AggregateRoot> = {
 	streamId: string;
 	payload: ISnapshot<A>;
 	aggregateName: string;
-	latest?: string;
+	latest: string | null;
```

Comment:
I could keep the same type here (I actually would rather keep it), but if I do I must change the e2e tests (line 81), where we check for null instead of undefined.